sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/QueryResult.cs

```diff
@@ -9,23 +9,47 @@
 
 namespace Azure.DigitalTwins.Core
 {
+    /// <summary>
+    /// The results of a query operation.
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
```

Comment:
Need to add a description here

Reply:
Linking to BasicDigitalTwin may be worthwhile if the user is querying twins, and not counts of twins or otherwise

Reply:
Just noticed this isn't public, but you can still add a small description for our own knowledge